core/common/assets/js/utils/helpers.js

```diff
@@ -20,9 +20,14 @@ export default class Helpers {
 	}
 
 	consoleWarn( ...args ) {
-		const style = `font-size: 12px; background-image: url("${ elementorCommon.config.urls.assets }images/logo-icon.png"); background-repeat: no-repeat; background-size: contain;`;
+		const style = `
```

Comment:
The code style is neccessary for the fix?

Reply:
No, will be removed.